### DISCUSSION TITLE
CSS `font-variant` and `font-feature-settings` values

### DISCUSSION THREAD

**Discussion by pawelurbanski** [rsms/inter#762] [Q&A]
Hello Everyone,
I've read numerous articles about font features and casually browsed Inter sources. I've checked the Inter features list using https://wakamaifondue.com/ to review the values.

I would like to get some clarification in the following areas:

1. Are the features listed eabled by default or I must enable them using `font-feature-settings` in the `@font-face` block?
2. Should I do the same with equivalent values using `font-variant` combining features?
3. Which of the features are duplicates and can be skipped?
4. Should I use `font-variant` for the body element in addition to the features in the `@font-face` block?
5. `font-variant` can have variants such as `font-vaiant-numeric` but is it allowed and valid to combine features in the generic `font-variant` such as `slashed-zero`?

I am willing to create the CSS for the Inter stylesheet or a jiist for future reference.

Thank you for answers and clarification...

All the best,
Pawel

**Comment by yisibl** [comment]
We should use `@font-feature-values` and `font-variant-alternates` instead of `font-feature-settings`.

`font-feature-settings` causes a lot of problems. https://issues.chromium.org/issues/324519293

https://github.com/Sorixelle/srxl.me/blob/4eb4f4a15cb2d21356df24c096d6a819cfdc1a99/public/fonts/inter/inter.css#L201-L222

**Comment by rsms** [comment]
[https://rsms.me/inter/inter.css](https://rsms.me/inter/inter.css) now has @font-feature-values declared, so you can customize features with e.g. `font-variant-alternates: character-variant(simplified-u, uc-g-with-spur)` instead of `font-feature-settings`
